web/src/components/TagList.tsx

```diff
@@ -83,6 +84,7 @@ const TagList = () => {
         </button>
       </div>
       <div className="flex flex-col justify-start items-start relative w-full h-auto flex-nowrap">
+        <TagItemContainer tag={untagged} tagQuery={filter.ignore} filterIgnore={true} />
```

Comment:
Because the `Uncategorized` is not a tag and won't be used very often, it shouldn't be shown on the top of the tag list. A better way to show this action is to add a dropdown. WDYT?

<img width="396" alt="image" src="https://github.com/usememos/memos/assets/24653555/f38e3ad8-058e-432f-88cd-e8dd9662041c">

Reply:
Yeah, that's valid. I'll work on it.